confChamber.py

```diff
@@ -43,20 +45,25 @@
 ohboard = getOHObject(options.slot,options.gtx,options.shelf)
 print 'opened connection'
 
+import os
 if options.gtx in chamber_vfatDACSettings.keys():
     print "Configuring VFATs with chamber_vfatDACSettings dictionary values"
-    parameters.defaultValues["IPreampIn"] = chamber_vfatDACSettings[options.gtx]["IPreampIn"]
-    parameters.defaultValues["IPreampFeed"] = chamber_vfatDACSettings[options.gtx]["IPreampFeed"]
-    parameters.defaultValues["IPreampOut"] = chamber_vfatDACSettings[options.gtx]["IPreampOut"]
-    parameters.defaultValues["IShaper"] = chamber_vfatDACSettings[options.gtx]["IShaper"]
-    parameters.defaultValues["IShaperFeed"] = chamber_vfatDACSettings[options.gtx]["IShaperFeed"]
-    parameters.defaultValues["IComp"] = chamber_vfatDACSettings[options.gtx]["IComp"]
+    for regName in chamber_vfatDACSettings[options.gtx].keys():
+        if regName not in parameters.defaultValues:
+            print("%s for link %d not recognized, please cross-check chamber_vfatDACSettings"%(regName, options.gtx))
+            print("list of possible values:", parameters.defaultValues.keys())
+            exit(os.EX_USAGE)
```

Comment:
are you sure you want to exit here? rather than just print the message and continue?

Reply:
If a `$USER` has specified a `regName` in the `chamber_vfatDACSettings` and it is not found in `parameters.defaultValues` then there configuration won't be what they are expecting.  

So I don't think printing and continuing will do much good, e.g. their configuration will still be wrong.  Additionally it might be really wrong and if other arguments are specified (e.g. `--vfatConfig`) any error message here will be buried in further output.

I think it would be more reliable to have this throw an error and be the last message printed to terminal (additionally this should be caught by `confAllChambers.py` as a problem).